elasticsearch/json-jvm-gc.yml

```diff
@@ -1,6 +1,6 @@
 type: json
 data:
-  url: /_nodes/_local/stats/indices,transport,http,thread_pool,jvm,process?format=smile
+  url: /_nodes/_local/stats/indices,transport,http,thread_pool,jvm,process,fs,script,discovery,ingest,adaptive_selection?format=smile
```

Comment:
Checked the call on older ES? Plus in a few other places, won't mention again.

Reply:
Yes, it seems to work on 2.4.4.